README.md

```diff
@@ -22,10 +22,22 @@ To do that, click on Affinidi logo in sidebar, then click on “Create an accoun
 
 ![Create an Account](./media/docs/create_account.png)
 
-When you’re authenticated, just click on “Create project” button and enter a project’s name. That’s it!  
-We’ll automatically create a digital identity for you – your personal DID.
+Once the account is created, a project named `Default Project` will be created automatically. As part of it, a digital identity will be created for you – your personal DID.
 
-![Project Details](./media/docs/project_details.png)
+![Default Project](./media/docs/default_project.png)
+
+Initially the `Default Project` will be set as your `Active Project`.
+
+Your `Active Project` is the project on which you can perform all the [operations](#features) below. There will always be only 1 active project at a time.
+
+You can also manually create projects by clicking on `Create Project` button and enter a project’s name.
+As the default project is the active project, all other projects will be shown under the `Inactive Projects` folder.
+
+![Inactive Projects](./media/docs/inactive_projects.png)
+
+You can quickly change the active project any time by clicking on the respective `Activate` icon.
+
+![Activate Project](./media/docs/activate_project.png)
```

Comment:
Personally, I think that **bold** here would make more sense than `monospace`, but this is nitpicking